bytecomp/bytelink.ml

```diff
@@ -577,7 +577,7 @@ let build_custom_runtime prim_name exec_name =
       [] in
   Ccomp.call_linker Ccomp.Exe exec_name
     (debug_prefix_map @ [prim_name] @ List.rev !Clflags.ccobjs @ [runtime_lib])
-    (Clflags.std_include_flag "-I" ^ " " ^ Config.bytecomp_c_libraries)
+    (Clflags.std_include_flag "-I" ^ " " ^ Config.bytecomp_c_libraries) = 0
```

Comment:
I would prefer a `let exitcode` binding here for readability, and same thing in the other diff below.